src/main/java/sungshin/sooon/controller/PostController.java

```diff
@@ -0,0 +1,95 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.dto.*;
+import sungshin.sooon.model.Account;
+import sungshin.sooon.model.CurrentUser;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.repository.PostRepository;
+import sungshin.sooon.service.PostService;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/post")
+public class PostController {
+
+    private final PostService postService;
+    private final PostRepository postRepository;
+
+    //게시글
+    @PostMapping("/create")
+    public Long create(@CurrentUser Account account, @RequestBody PostCreateRequestDto dto){
+        if(account != null){
+            dto.setAccount(account);
+        }
+        return postService.save(dto);
+    }
+
+    @GetMapping("/{post_id}")
+    public PostResponseDto read(@PathVariable Long post_id){
+        return postService.read(post_id);
+    }
+
+    @PutMapping("/{post_id}")
+    public Long update(@PathVariable Long post_id, @RequestBody PostUpdateRequestDto dto){
+        return postService.update(post_id, dto);
+    }
+
+    @GetMapping("/{post_id}/delete")
```

Comment:
2. GET 메서드와 쿼리 파라미터로 상태를 변경하면 안된다
상태를 변경하려면 GET 메서드 대신 PUT, POST, DELETE 메서드를 사용하라.

출처: https://multifrontgarden.tistory.com/252 [우리집앞마당]
```suggestion
    @DeleteMapping("/{post_id}")
```